paasta_tools/kubernetes_tools.py

```diff
@@ -1173,6 +1178,16 @@ def get_kubernetes_environment(self) -> List[V1EnvVar]:
                     field_ref=V1ObjectFieldSelector(field_path="spec.nodeName")
                 ),
             ),
+            V1EnvVar(
+                name="PAASTA_CLUSTER",
+                value_from=V1EnvVarSource(
+                    field_ref=V1ObjectFieldSelector(
+                        field_path="metadata.labels['"
+                        + paasta_prefixed("cluster")
+                        + "']"
```

Comment:
i'm assuming we're not using an f-string here to get around having to deal with a mess of escaping quotes?

Reply:
yeah, i feel like it's somewhat readable this way at least.